config/locales/doc_auth/es.yml

```diff
@@ -211,9 +211,9 @@ es:
       text_message: Enviamos un mensaje a su teléfono
       upload_from_computer: Continuar en esta computadora
       upload_from_phone: Utilice su teléfono para tomar las fotos
-      verify_at_post_office: Verifique su identidad en una oficina de correos
+      verify_at_post_office: Verifique su identidad en una oficina de correos.
```

Comment:
There is a period here- but not in the French translation for `verify_at_post_office`. (I did not check English yet.) I would work with design team to see what should and what should not have a period for all of your updated. I think they should be consistent for each language.